rocAL/rocAL/source/api/rocal_api_info.cpp

```diff
@@ -121,9 +124,9 @@ TimingInfo
     auto context = static_cast<Context *>(p_context);
     auto info = context->timing();
     // INFO("bbencode time "+ TOSTR(info.bb_process_time)); //to display time taken for bbox encoder
-    if (context->master_graph->is_video_loader())
-        return {info.video_read_time, info.video_decode_time, info.video_process_time, info.copy_to_output};
-    else
+    // if (context->master_graph->is_video_loader()) // TO BE FIXED
```

Comment:
make sure to have this feature

Reply:
We will fix this in future PR

Reply:
In tensor pipeline we have a single output routine for image and video.
is_video_loader flag has been removed.
We will fix this in future PR by adding only one timers for decode_time, read_time and process_time. Instead of having separete timers for each as image_read_time, video_read_time.

Reply:
@fiona-gladwin In that case, please change name from image_read_time to "read_time" in future PR

Reply:
Sure Lakshmi, We will be adding the change in subsequent PR